graphql_base/types.py

```diff
@@ -41,3 +49,33 @@ def __init_subclass_with_meta__(cls, default_resolver=None, **options):
         return super(OdooObjectType, cls).__init_subclass_with_meta__(
             default_resolver=default_resolver, **options
         )
+
+
+def alias(target_name):
+    """Return a valid resolver that aliases to another attribute name.
+
+    This lets you work around unsuitable Odoo field names without
+    boilerplate and without manually replicating what the default
+    resolver does.
+
+    Instead of::
+
+        @staticmethod
+        def resolve_child(root, info):
+            return root.child_id or None
+
+    Write::
+
+        resolve_child = alias("child_id")
+    """
+
+    @classmethod
+    def resolve_field(cls, root, info):
+        resolver = (
+            cls._meta.default_resolver or graphene.types.resolver.get_default_resolver()
+        )
+        field = cls._meta.fields.get(info.field_name, None)
+        default_value = getattr(field, "default_value", None)
+        return resolver(target_name, default_value, root, info)
```

Comment:
Could this be simplified to something like this?

```suggestion
    @staticmethod
    def resolve_field(root, info):
        return getattr(root, target_name) or None
```

Reply:
It copies the way graphene looks for values, so you still get all the finicky things `odoo_attr_resolver` does by default. And if you use your own object type with other custom behavior then that's handled too.

`getattr(root, target_name) or None` would work for the example in the docstring, but not for datetimes, 2many relations, and so on.